src/Card.js

```diff
@@ -1,31 +1,56 @@
 import { string } from "prop-types";
 
-const Card = ({color, status}) => (
+const Card = ({color, status}) => {
 
-    <div style={{ backgroundColor: color }}>
-        <span>Estado: {status}</span>
+    function showText () {
 
-        {status === 'procesar' ? (
-            <p>Tiene una tarea por procesar.</p>
-        ) : status === 'procesando' ? (
-            <>
-                <div>
-                    <p>Tareas:</p>
-                    <ul>
-                        <li>HTML</li>
-                        <li>JavaScript</li>
-                        <li>CSS</li>
-                    </ul>
-                </div>
-                <p>La tarea se esta procesando.</p>
-            </>
-        ) : status === 'finalizado' ?(
-            <p>La tarea ha finalizado</p>
-        ) : (
-            alert('Un error ha ocurrido')
-        )}
-    </div>
-);
+        if (status === 'procesar') {
+
+            return <p>Tiene una tarea por porcesar</p>;
+        }
+
+        if (status === 'procesando') {
+
+            const list = [
+                {title: 'HTML', id: 1},
+                {title: 'CSS', id: 2},
+                {title: 'JavaScript', id: 3},
+                {title: 'Java', id: 4},
+                {title: 'JSX', id: 5}
+            ];
+
+            const listItems = list.map (items =>
+                <li key={items.id}>
+                    {items.title}
+                </li>
```

Comment:
Solo una sugerencia, items es prural, deberia ser item que es singular:
```suggestion
            const listItems = list.map (item =>
                <li key={item.id}>
                    {item.title}
                </li>
```